src/integrationtests/java/com/aws/greengrass/integrationtests/deployment/DeploymentTaskIntegrationTest.java

```diff
@@ -1246,6 +1246,38 @@ void GIVEN_services_running_WHEN_new_deployment_asks_to_skip_update_policy_check
         assertEquals(DeploymentResult.DeploymentStatus.SUCCESSFUL, result.getDeploymentStatus());
     }
 
+    @Test
+    @Order(101)
+    void GIVEN_services_with_dependency_broken_WHEN_new_deployment_failure_handling_policy_rollback_THEN_deployment_rollback_is_successful(
+            ExtensionContext context) throws Exception {
+
+        ignoreExceptionUltimateCauseOfType(context, ServiceUpdateException.class);
+
+        // Deploy a broken config with no rollback
+        Future<DeploymentResult> resultFuture = submitSampleJobDocument(
+                DeploymentTaskIntegrationTest.class.getResource("DependencyDoNothing.json").toURI(),
+                System.currentTimeMillis());
+        DeploymentResult result = resultFuture.get(DEPLOYMENT_TIMEOUT, TimeUnit.SECONDS);
+        resultFuture.get(60, TimeUnit.SECONDS);
```

Comment:
does it need 60s? can this be shorter?

Reply:
60s is shortened from default DEPLOYMENT_TIMEOUT(80s), I prefer to stay as it since it's just a timeout and won't effect the actual test execution time.